web-ts/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "web-ts",
+  "version": "0.1.0",
+  "private": true,
+  "engines": {
+    "node": ">=6.9",
+    "npm": ">=3.8"
+  },
+  "scripts": {
+    "start": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=integreat-test-cms --env.dev_server",
+    "start:malte": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=malte --env.dev_server",
+    "start:aschaffenburg": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=aschaffenburg --env.dev_server",
+    "build": "yarn build:integreat",
+    "build:integreat": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=integreat",
+    "build:malte": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=malte",
+    "build:aschaffenburg": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=aschaffenburg",
+    "build:integreat-test-cms": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=integreat-test-cms",
```

Comment:
This can be simplified by creating a script:

```
"webpack-ts": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts",
"build:integreat-test-cms": "yarn run webpack.ts --config...."
```